package.json

```diff
@@ -4,9 +4,10 @@
   "description": "Bindings for WebThings Gateway add-ons",
   "scripts": {
     "prepare": "if [ ! -d schema/messages ]; then git submodule init; git submodule update; fi",
```

Comment:
I am not sure how publishing to npm is executed for this package, but you might want to compile to JS on `prepare` hook. See https://docs.npmjs.com/cli/v6/using-npm/scripts/#prepare-and-prepublish → Use Cases.

Reply:
I wonder if the dev dependencies are available in this step.
I'll try it out after the type generation step is finished.

Reply:
Done